src/rust/engine/src/context.rs

```diff
@@ -249,7 +264,35 @@ impl Core {
     }
     let graph = Arc::new(InvalidatableGraph(Graph::new()));
 
-    let http_client = reqwest::Client::new();
+    // These certs are for downloads, not to be confused with the ones used for remoting.
+    let ca_certs = if let Some(ref path) = ca_certs_path {
+      let mut content = String::new();
+      std::fs::File::open(path)
+        .and_then(|mut f| f.read_to_string(&mut content))
+        .map_err(|err| format!("Error reading root CA certs file {:?}: {}", path, err))?;
```

Comment:
`path` here is of type `PathBuf`, which has a special method
https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.display

used to get a safe string representation of a possibly-non-utf8 file system path